yt/chyt/controller/internal/jupyt/controller.go

```diff
@@ -45,12 +39,10 @@ func (c *Controller) UpdateState() (changed bool, err error) {
 }
 
 func (c *Controller) buildCommand(speclet *Speclet) (command string, env map[string]string) {
-	// TODO(max): take port from YT_PORT_0.
 	// TODO(max): come up with a solution how to pass secrets (token or password) without exposing them in the
 	// strawberry attributes.
 
-	cmd := fmt.Sprintf(
-		"bash -x start.sh /opt/conda/bin/jupyter lab --ip '*' --port %v --LabApp.token='' --allow-root >&2", JupytPort)
+	cmd := "bash -x start.sh /opt/conda/bin/jupyter lab --ip '*' --port $YT_PORT_0 --LabApp.token='' --allow-root >&2"
```

Comment:
I'm not sure this will work. I've checked only `yt vanilla --tasks '{task={job_count=1; command="echo \$YT_PORT_0 >&2"; port_count=1; cpu_limit=1};}' --spec '{resource_limits={user_slots=1}}'`

Reply:
it works